scripts/python/venv-setup-prep.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="$SCRIPT_DIR/../.."
+
+# Imports
+source "$ROOT_DIR/scripts/utils/log.sh"
+
+# -o errexit: abort script if one command fails
+# -o errtrace: the ERR trap is inherited by shell functions
+# -o pipefail: entire command fails if pipe fails
+set -o errexit -o errtrace -o pipefail
+
+# Print usage options for this script.
+print_help() {
+  cat <<EOF
+Usage: $(basename "${0}") [OPTIONS]
+
+Options:
+  -h, --help                      Display this text.
+  --venv-pth <venv-dir> <paths>   Add path configuration files to the specified venv (similar to extending PYTHONPATH)
+                                  hint: <paths> is a ':' separated string list.
+
+Examples:
+  $(basename "${0}") --venv-pth
+EOF
+}
+
+# Parse arguments.
+parse_args() {
+  while test $# -gt 0; do
+    arg="$1"
+    case "$arg" in
+    --venv-pth)
```

Comment:
For the sake of clarity.

```suggestion
    --venv-path)
```

Reply:
hmm no it's not quite just a path, PTH are actually files, let's keep it as is